backend/cabin-sorting/service/sortedHackers-service.ts

```diff
@@ -0,0 +1,24 @@
+import * as sortedHackersDao from "../dao/sortedHackers-dao.js";
+
+const getSortedHackers = async () => {
+    const rawHackerData = await sortedHackersDao.getSortedHackers();
+    const sortedHackers = rawHackerData.map(hackerData => {
+      const applicationResponses = hackerData.applicationResponses;
+      Object.keys(applicationResponses).forEach((key, keyIndex) => {
+        applicationResponses["question" + keyIndex] = applicationResponses[key]
+        delete applicationResponses[key]
+      })
+      return { ...applicationResponses,
+        id: hackerData._id,
+        email: hackerData.email
+      }
+    })
+    return sortedHackers;
+};
```

Comment:
I think we can clean this method up! Try using the map to transform the hackerData objects and reduce() instead of a for each loop to update the responses.